app/src/main/java/com/example/trainingintive/util/BindingAdapters.kt

```diff
@@ -12,3 +15,11 @@ fun bindImage(imgView: ImageView, imgUrl: String?) {
         imgView.load(imgUri)
     }
 }
+
+@BindingAdapter("data")
+fun bindDataToRecyclerView(recyclerView: RecyclerView, items: List<ActivityModel>) {
```

Comment:
Minor: It could be an extension to `RecyclerView` to make it more "Kotlinish" :)

Reply:
It will also shorten this `BindingAdapter` because you will use `RecyclerView` instance as a `this`.

Reply:
you right, extension is shorter :)